### DISCUSSION TITLE
宝塔缓存问题

### DISCUSSION THREAD

**Discussion by Cyrene963** [coaidev/coai#299] [Q&A | 问题解答] [answered]
[//]: # (方框内删除已有的空格，填 x 号)
+ [] 我已确认目前没有类似 issue
+ [] 我已确认我已升级到最新版本
+ [] 我已完整浏览项目 README 和项目文档并未找到解决方案
+ [] 我理解并愿意跟进此 issue，协助测试和提供反馈
+ [] 我将以礼貌和尊重的态度提问，不得使用不文明用语 (包括在此发布评论的所有人同样适用, 不遵守的人将被 block)
+ [] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**问题描述**
渠道设置明明删除了该渠道在界面依然显示存在，且刷新后也无效。后续新增渠道也不予显示了。
**复现步骤**
删除渠道后出现bug，如上。
**预期结果**
成功一次性删除
**日志信息**

**相关截图 (如果有)**
![image](https://github.com/user-attachments/assets/a8ca3f9d-4a9a-45ce-a05f-4df99f4334b6)

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
无法复现，F12看一下请求返回

[a comment by an automated account is not shown]

**Comment by Cyrene963** [comment]
> 无法复现，F12看一下请求返回

![image](https://github.com/user-attachments/assets/007f286f-08d5-48d5-af47-a14774171297)
``
{message: "conversation not found", status: false}
message
: 
"conversation not found"
status
: 
false
``

**Comment by Cyrene963** [comment]
> 无法复现，F12看一下请求返回

已经删除了 但是还是显示的状态 还有 我聊天界面是用户共享的吗？怎么我换了个账号界面聊天记录还是意义一样的

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment, marked as answer]
我根据你这两个现象，我猜你是不是开宝塔强制缓存了。
不要开缓存，内部就有redis设计，nginx开缓存会把之前的接口也缓存了，因为不是cookie存储token (用jwt+localstorage)，因此不在http中传递， 宝塔的nginx缓存配置没法识别请求是哪个用户的， 不要乱配置缓存。

解决方案：关闭缓存

**Reply by Cyrene963** [reply]
感谢感谢 第一次玩反向代理配置这些不是很懂

[a comment by an automated account is not shown]
